department-of-veterans-affairs/abd-vro
# Local Docker Build Files for Python Services

## What was the problem?

When you run `gradle build`, the `build.gradle` file copies over (*.py files from `service-python` and all files in `service-python/<whatever_service>/src` ) into your service's `build` folder. Gradle sets it up so all files are within the same directory level
Since the Dockerfile is located in `service-python/<whatever_service>/docker` , when you try to build it manually, it can't find the `requirements.txt` since it doesnt exist.
The Dockerfile only knows of what's inside the folder it gets called from, so you can't do a docker build from within the `service-python/<whatever_service>/src/docker`

## How does this fix it?[^1]

Create new Dockerfiles that will build the docker image without the need for Gradle.

## How to test this PR
- Navigate to `service-python` and run `docker build -t localtest -f Dockerfile.local.assessclaimdc6602 .` or `docker build -t <tag> -f <new dockerfile> .`
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
